rfc6962-bis.xml

```diff
@@ -614,6 +607,9 @@ labels extension.
 protocol to which the SCT conforms. This version is v2. Note that <xref target="RFC6962">SignedCertificateTimestamp v1</xref> had a different definition of <spanx style="verb">signed_entry</spanx>.
         </t>
         <t>
+          <spanx style="verb">log_id</spanx> is the DER encoding of an OID, excluding the ASN.1 tag and length bytes, that the log operator has allocated for the purpose of uniquely identifying this log. This OID is specified in the log's metadata.
```

Comment:
Dropping the ASN.1 tag and length bytes implies the length byte would be fixed, correct?